TheaterSchedule.BLL/Services/ScheduleServices.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TheaterSchedule.DAL;
+using TheaterSchedule.DAL.Entities;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.BLL
+{
+    public class ScheduleServices : IScheduleServices
+    {
+        private ITheaterScheduleUnitOfWork uow;
+
+        public ScheduleServices(ITheaterScheduleUnitOfWork repo)
+        {
+            uow = repo;
+        }
+
+        public IEnumerable<Schedule> GetListPerformancesByDateRange(DateTime? startDate, DateTime? endDate)
+        {
+            IEnumerable<Schedule> listPerfomances = uow.Schedule
+                .GetWithInclude(p => p.performance)
+                .Where(p => p.Beginning >= startDate && p.Beginning <= endDate);
```

Comment:
Handle case when dates are null 
like this
` (!endDate.HasValue || p.Beginning <= endDate)`

this select logic looks like repository method. But into repository and just call in service